scipy/stats/_multivariate.py

```diff
@@ -960,6 +964,36 @@ def rvs(self, mean=None, rowcov=1, colcov=1, size=1, random_state=None):
             out = out.reshape(mean.shape)
         return out
 
+    def equivalent_multivariate_normal(self, mean=None, rowcov=1, colcov=1,
+                                       random_state=None):
```

Comment:
Oh, I think I must have mis-read or mis-understood the previous discussion. I thought someone had requested this method. I agree too. Scrap this.